### ISSUE TITLE
OneGet package provider

### ISSUE THREAD

**Issue by m5x** [0install/0install-dotnet#29]
Hi, I noticed there is an implementation of OneGet package provider in 0install-win repo but not here. Is there something blocking it?

**Comment by bastianeicher** [comment]
Hi @m5x,

thanks for your interest in Zero Install and its OneGet provider.

0install-win (Zero Install for Windows) uses 0install-dotnet (cross-platform core of Zero Install implemented in .NET) as a dependency and extends it with a GUI and various OS-specific integrations.

Since OneGet is Windows-only it wouldn't really make sense to provide a OneGet provider for the cross-platform part of Zero Install if there is already one for the Windows-specific superset.

Do you have some other use-case not covered by the 0install-win OneGet provider (e.g. using Windows Nano Server)?

Regards
Bastian

**Comment by m5x** [comment]
I thought OneGet is cross-platform for a few years now that is part of PowerShell Core.

I found 0install while looking for existing approaches to distribution of portable software. As a part of an in-house dev-ops automation effort I need to be able to automate installation and updating of custom software that cannot be installed through OS-provided package manager.

0install ticks a lot of boxes and I also like the fact that it has a nicely written .NET Core implementation. Cross-platform PowerShell integration and package provider would be another big plus because I am seriously considering using PowerShell for all the automation on both Linux and Windows now.

Currently I decided to explore a bit different route because I do not need to care about meta files, dependencies, licenses, hosting, etc. and it is enough for me to be just able to copy/delta update selected directories from central repository to multiple devices and run a script afterwards. However I really liked what I saw while exploring your .NET implementation and might as well go back to 0install at some point in the future when I have more suitable use case for it. Good work!
